front/cypress/e2e/admin/input_manager.cy.ts

```diff
@@ -61,10 +67,10 @@ describe('Input manager', () => {
 
             // do a refresh for the new idea to appear
             cy.visit('/admin/ideas/');
-            // grab and open assignee filter menu
-            cy.get('#e2e-select-assignee-filter').click();
             // click on Assigned to me filter
-            cy.get('#e2e-assignee-filter-assigned-to-user').click();
+            const optionLabelText = 'Assigned to me';
+            selectAssigneeFilter(optionLabelText);
+            checkSelectedAssigneeFilter(optionLabelText);
```

Comment:
I notice we're using text rather than checking IDs now - was there a specific reason for this? :)

Reply:
Actually nevermind! I think I know why. Since the option object is different - semantic UI supports an ID, but our component library one does not :+1:

Reply:
Might be nice to add that into our component library Select component at some point though if not now :thinking: Certainly useful for the E2E tests.

Reply:
I followed [the Cypress docs](https://docs.cypress.io/api/commands/select). :) You should pass the text/value/index of the option you'd like to check. So alternatively I could use `value`. However, these filter names haven't changed in years. And when they do, fixing them might take about the same amount of time it'll take me to change everything to use value, rerun the tests, have another review, etc. Plus, there's a chance that when the filter names change, the values will have changed as well and the test would have to get fixed regardless. 😄

TL;DR: next time I might go for value, but taking the context in mind here, I'll stick with this.